index.html

```diff
@@ -167,7 +167,7 @@ <h1 class="heading"><i class="fas fa-cloud"></i> sButton CDN</h1>
         <div class="instruction row">
           <div class="desc col col-md-6 col-sm-12">
             When you only need to include sButtons’s compiled CSS , you can use
-            SButtonCDN.
+            SButton CDN.
```

Comment:
Change SButton to sButtons and a link to it.

Reply:
I have to add this link - ("https://cdn.jsdelivr.net/gh/sbuttons/sbuttons/dist/sbuttons.css") to "sButton CDN" in heading or in subsection ?

Reply:
in the subsection and in the code block

Reply:
ok